UNRELEASED.md

```diff
@@ -0,0 +1,14 @@
+**Breaking:**
+
+## bpk-component-rating:
+ > Currently, the implementation of Rating component in backpack is based on the deprecated design, which means it's not used for any public facing microsites, such as Hotels website, Banana, Hercules, etc. Therefore, we'd like to refresh `bpk-component-rating` to realize a new design in the backpack, so that we can maintain a consistent traveller experience of the hotels product in each verticals.
```

Comment:
Can this comment be removed as it won't carry through to the changelog when we release and we shouldn't mention these things here

Reply:
Make sense, these comments have been removed.